docs/sphinx/source/menu/SofaPlugin.rst

```diff
@@ -324,155 +410,244 @@ Congratulations! You made your first SOFA scene in Python3!
+        print(finalTime)
+
+Note that:
+* accessing the Data "time" doing ``root.time`` would only return the python pointer and not the value of the Data
+* Data which are vectors can be casted as numpy arrays
 
-.. code-block:: python
 
-	# Required import for python
-	import Sofa.Core
-	import Sofa.Simulation
-	import SofaRuntime
 
+Write access
+^^^^^^^^^^^^
 
-	# Function creating the scene and describing the graph (with nodes and components) before it is initialized
-	def createScene(root):
-		return root
+In the same way, Data can be modified (write access) using the ``.value`` accessor. Here is an example (without GUI) computing 10 time steps, then setting the world gravity to zero and recomputing 10 time steps:
+
+
+.. code-block:: python
 
 
-	#Main function ONLY called if this script is called from a python environment
 	def main():
 
-		# Make sure to load all SOFA libraries and plugins
-		SofaRuntime.importPlugin("Sofa.Component.StateContainer")
+        # Call the SOFA function to create the root node
+        root = Sofa.Core.Node("root")
 
-		# Generate the root node
-		root = Sofa.Core.Node("root")
+        # Call the createScene function, as runSofa does
+        createScene(root)
 
-		# Call the above function to create the scene graph
-		createScene(root)
+        # Once defined, initialization of the scene graph
+        Sofa.Simulation.init(root)
 
-		# Once defined, initialization of the scene graph
-		Sofa.Simulation.init(root)
+        # Run the simulation for 10 steps
+        for iteration in range(10):
+                Sofa.Simulation.animate(root, root.dt.value)
+        
+        # Print the position of the falling sphere
+        print(root.sphere.mstate.position.value)
 
-		# Run the simulation for 10 steps
-		for iteration in range(10):
-			Sofa.Simulation.animate(root, root.dt.value)
+        # Increase the gravity
+        root.gravity.value = [0, 0, 0]
 
+        # Run the simulation for 10 steps MORE
+        for iteration in range(10):
+                Sofa.Simulation.animate(root, root.dt.value)
 
-	# Function used only if this script is called from a python environment, triggers the main()
-	if __name__ == '__main__':
-	    main()
+        # Print the position of the falling sphere
+        print(root.sphere.mstate.position.value)
+
+
+The ``.value`` accessor works for simple Data structures such as a string, an integer, a floating-point numbers or a vector of these.
+
+For more complex Data such as Data related to the degrees of freedom (e.g. Coord/Deriv, VecCoord/VecDeriv), the ``.writeableArray()`` write accessor must be used. Let's consider a scene graph that would have a *ConstantForceField* named "CFF" in the sphere node, and that we would like to modify the Data "totalForce" (a Deriv defined in `ConstantForceField.h <https://github.com/sofa-framework/sofa/blob/master/Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/ConstantForceField.h#L66>`_), we should then write something like:
```

Comment:
Explain why using the writeableArray() is better to value. Here explaining because it is "complex" is fuzzy.